src/components/Footer.jsx

```diff
@@ -1,5 +1,52 @@
+import whiteLogo from "../../public/assets/whiteLogo.svg";
+import Image from "next/image";
+import { ICONS } from "@/data/icons";
+
 const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <div className="w-full bg-ewb-green">
+      <p className="leading-4"> &nbsp; </p>
+
+      <div className="w-full bg-ewb-blue-200 border-t-8 border-ewb-green">
```

Comment:
make this the parent component.
think of the entire footer as two parts: the logo at the left and the information at the right. You can add a "flex justify-between at the parent component